command : ssh fsd319 cat script.py | python -
result : 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'U' is not defined

Return the error of the python script. You're welcome.
